sotodlib/toast/ops/mlmapmaker.py

```diff
@@ -163,12 +185,15 @@ class MLMapmaker(Operator):
     weather = Unicode("vacuum", help="Weather to assume when making maps")
     site    = Unicode("so",     help="Site to use when making maps")
 
-    maxiter = Int(500, help="Maximum number of CG iterations")
+    maxiter = List(
+        [500],
+        help="Maximum number of CG iterations. Use commas to control each pass separately.",
```

Comment:
Maybe change the help string to explicitly say, "List of maximum number of CG iterations for each pass".

Reply:
Good point. Changed.